toolkit/scripts/toolchain.mk

```diff
@@ -70,8 +78,8 @@ clean-toolchain-rpms:
 	for f in $(toolchain_rpms_noarch); do rm -vf $(RPMS_DIR)/noarch/$$f; done
 
 copy-toolchain-rpms:
```

Comment:
I'm curious if this now could just depend on `$(toolchain_out_rpms)` and we could remove the whole logic below?

Reply:
I was debating what to do with copy-toolchain-rpms, I decided on leaving it unchanged since it could break a lot of pipelines if it went wrong.

If we want to do that I can move `$(toolchain_out_rpms)` out of the `ifeq ($(REBUILD_TOOLCHAIN),y)` block and only leave `$(RPMS_DIR): $(toolchain_out_rpms)` in the conditional.